docs/library/bluetooth.rst

```diff
@@ -759,3 +760,38 @@ Constructor
 
     - A 16-bit integer. e.g. ``0x2908``.
     - A 128-bit UUID string. e.g. ``'6E400001-B5A3-F393-E0A9-E50E24DCCA9E'``.
+
+
+Constants
+---------
+
+.. data:: FLAG_READ
```

Comment:
There's actually a TODO in the .c file about these constants:
```
// TODO: Deprecate these flags (recommend copying the constants from modbluetooth.h instead).
```
So if they are to be documented here, there needs to be a comment here saying something like "The following flags are deprecated.  Instead copy the constants from `modbluetooth.h`, along with the other constants defined there."

Also the descriptions below read like they correspond to events.  But the flags correspond to attributes of characteristics, used and described by `BLE.gatts_register_services`